question/import_form.php

```diff
@@ -34,10 +34,15 @@
  *
  * @copyright  2007 Jamie Pratt me@jamiep.org
  * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ * @deprecated since Moodle 4.0 MDL-71608
+ * @see qbank_importquestions\forms\question_import_form
```

Comment:
please correct the namespace here
i believe its qbank_importquestions\form\question_import_form